install.sh

```diff
@@ -110,7 +110,7 @@ update_profile() {
    if [[ -n "$PROFILE_FILE" ]]; then
      if ! grep -q "\.turso" $PROFILE_FILE; then
         printf "\n${bright_blue}Updating profile ${reset}$PROFILE_FILE\n"
-        printf "\n# Turso\nexport PATH=\"$INSTALL_DIRECTORY:\$PATH\"\n" >> $PROFILE_FILE
+        printf "\n# Turso\nexport PATH=\"\$PATH:$INSTALL_DIRECTORY\"\n" >> $PROFILE_FILE
```

Comment:
Do we need `\` before `$`?

Reply:
Yes, to escape it. Otherwise it injects your actual PATH into the string.